tests/clock/test_clock.py

```diff
@@ -0,0 +1,427 @@
+import logging
+import random
+import string
+import pytest
+import time
+import allure
+import datetime as dt
+
+from contextlib import contextmanager
+from tests.common.errors import RunAnsibleModuleFail
+
+pytestmark = [
+    pytest.mark.topology('any'),
+    pytest.mark.sanity_check(skip_sanity=True),
+    pytest.mark.disable_loganalyzer,
+    pytest.mark.skip_check_dut_health,
+    pytest.mark.clock
+]
+
+
+@contextmanager
+def allure_step(step_msg):
+    """
+    @summary:
+        Context manager that wraps allure step context and a log with the same message
+    @param step_msg: The desired step message
+    """
+    with allure.step(step_msg) as allure_step_context:
+        logging.info('Step start: {}'.format(step_msg))
+        try:
+            yield allure_step_context
+        finally:
+            logging.info('Step end: {}'.format(step_msg))
+
+
+class ClockConsts:
+    STDOUT = "stdout"
```

Comment:
don't see any reason to define those as constants

Reply:
@roy-sror  , which of these u mean to?

Reply:
gotit, removed stdout and stdin constants